### PR TITLE
Initial support for iPhone & iPad.

### PR DESCRIPTION
**Pull request by philmoz**

### CODE REVIEW

File: focuspoints.lrdevplugin/AppleDelegates.lua [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+require "Utils"
+
+AppleDelegates = {}
+
+--[[
+-- metaData - the metadata as read by exiftool
+--]]
+function AppleDelegates.getAfPoints(photo, metaData)
+
+  local imageWidth = ExifUtils.findFirstMatchingValue(metaData, { "Image Width", "Exif Image Width" })
```
Comment (musselwhizzle) [inline focuspoints.lrdevplugin/AppleDelegates.lua]: using ExifUtils but there's no  ```require ExifUtils```

File: focuspoints.lrdevplugin/AppleDelegates.lua [hunk 2]
```diff
@@ -0,0 +1,70 @@
+--[[
+  Copyright 2017 philmoz & Joshua Musselwhite, Whizzbang Inc
```
Comment (musselwhizzle) [inline focuspoints.lrdevplugin/AppleDelegates.lua]: I actually don't know enough about licensing to know if this is legit. I'm not sure if the license needs to stay the same or not. Looking through other Github repos, I haven't found any where individual contributors are listed in the license at a later time. I need to look more into this.

**Comment by musselwhizzle** [comment]
that's really cool!

File: focuspoints.lrdevplugin/AppleDelegates.lua (hunk 1 above)
Comment (philmoz) [inline focuspoints.lrdevplugin/AppleDelegates.lua]: ExifUtils is loaded by DefaultPointRenderer, which is loaded by PointsRendererFactory before the other delegates are loaded.

Comment (musselwhizzle) [inline focuspoints.lrdevplugin/AppleDelegates.lua]: Understood. Would still like to see each needed called out explicitly. The internals of the other classes could change breaking this file.

File: focuspoints.lrdevplugin/AppleDelegates.lua (hunk 2 above)
Comment (musselwhizzle) [inline focuspoints.lrdevplugin/AppleDelegates.lua]: Due to my lack of lawyer'ign and from what I'm seeing, I'm going to request that the  license stays as-is at this time. Legals suggest a Contributor License Agreement....which I see nobody else on github using.

File: focuspoints.lrdevplugin/AppleDelegates.lua (hunk 1 above)
Comment (philmoz) [inline focuspoints.lrdevplugin/AppleDelegates.lua]: None of the delegate files requires ExifUtils explicitly?

Comment (musselwhizzle) [inline focuspoints.lrdevplugin/AppleDelegates.lua]: Bugs :)

File: focuspoints.lrdevplugin/AppleDelegates.lua (hunk 2 above)
Comment (philmoz) [inline focuspoints.lrdevplugin/AppleDelegates.lua]: I think you're confusing copyright and license.

Comment (musselwhizzle) [inline focuspoints.lrdevplugin/AppleDelegates.lua]: maybe. But for now, this is my request if it's to get merged.

**Pull request merged**